WallGenerator.java

```diff
@@ -0,0 +1,56 @@
+package brickBreaker;
+
+import java.awt.BasicStroke;
+import java.awt.Color;
+import java.awt.Graphics2D;
+
+public class WallGenerator {
+    private int wall[][];
+    private int brickWidth;
+    private int brickHeight;
+    public WallGenerator(int row, int col) {
+        setWall(new int[row][col]);
```

Comment:
Setters/getters are only used to update/get private members of another class. So, you don't need to use setters and getters in the same class (like done here). But it would be helpful in @Aiswarya-anand's `Gameplay` class :D